routes/_testCommon.js

```diff
@@ -3,15 +3,19 @@
 const db = require("../db.js");
 const User = require("../models/user");
 const Company = require("../models/company");
+const Job = require("../models/job");
 const { createToken } = require("../helpers/tokens");
-const { create } = require("../models/company");
+const { JsonWebTokenError } = require("jsonwebtoken");
```

Comment:
Is this being used anywhere?

Reply:
Nope, had it for a reason, realized I didn't need it there. Will remove.